diamond/diamond.conf

```diff
@@ -21,7 +21,7 @@ group = nogroup
 pid_file = /var/run/diamond.pid
```

Comment:
I updated this file, as it was not really according to the one in the container.